internal/provisioner/sloth.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package provisioner
+
+import (
+	"context"
+	"time"
+
+	"github.com/mattermost/mattermost-cloud/k8s"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	slothv1 "github.com/slok/sloth/pkg/kubernetes/api/sloth/v1"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func createPrometheusServiceLevel(psl slothv1.PrometheusServiceLevel, k8sClient *k8s.KubeClient, logger log.FieldLogger) error {
+	wait := 60
```

Comment:
Looks like we do not need this as a variable. Or may be `wait := time.Second * 60`. Your call. 0/5

Reply:
I've changed it to a default wait time using a duration instead of an int 👍